docs/integrations/sources/instagram.md

```diff
@@ -0,0 +1,74 @@
+# Instagram
+
+## Sync overview
+
+This source can sync data for the Instagram Business Account available in the Facebook Graph API: User, Media, and Stories. It can also sync Media/Story and User Insights.
+
+### Output schema
+
+This Source is capable of syncing the following core Streams:
+
+* [User](https://developers.facebook.com/docs/instagram-api/reference/ig-user)
+  * [User Insights](https://developers.facebook.com/docs/instagram-api/reference/ig-user/insights)
+* [Media](https://developers.facebook.com/docs/instagram-api/reference/ig-user/media)
+  * [Media Insights](https://developers.facebook.com/docs/instagram-api/reference/ig-media/insights)
+* [Stories](https://developers.facebook.com/docs/instagram-api/reference/ig-user/stories/)
+  * [Story Insights](https://developers.facebook.com/docs/instagram-api/reference/ig-media/insights)
+
+For more information, see the [Instagram API ](https://developers.facebook.com/docs/instagram-api/) and [Instagram Insights API documentation](https://developers.facebook.com/docs/instagram-api/guides/insights/).
+
+### Data type mapping
+
+| Integration Type | Airbyte Type | Notes |
+| :--- | :--- | :--- |
+| `string` | `string` |  |
+| `number` | `number` |  |
+| `array` | `array` |  |
+| `object` | `object` |  |
+
+### Features
+
+| Feature | Supported?\(Yes/No\) | Notes |
+| :--- | :--- | :--- |
+| Full Refresh Sync | Yes |  |
+| Incremental Sync | Yes | only User Insights |
+
+### Rate Limiting & Performance Considerations
+
+Instagram, like all Facebook services, has a limit on the number of requests.
```

Comment:
Can you mention that the connector gracefully handles rate limiting?

Reply:
done